config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -432,62 +432,12 @@ presubmits:
         - --use-logexporter
         image: gcr.io/k8s-testimages/kubekins-e2e:v20200901-eeeadc5-master
         resources:
+          limits:
+            cpu: 2
+            memory: "8Gi"
```

Comment:
The reason to have 8Gi here and 6Gi for periodic is that we need more for bazel?

Reply:
Bazel is not used here to build K8s - it's perf-tests presubmit.

When filling this value I've done manual inspection of Metrics Explorer and just rounded it to 8Gi instead of 6Gi.

Changing it to 6Gi to match periodic.